dockerfiles/init/manifests/codenvy.pp

```diff
@@ -303,6 +303,12 @@
   $db_schema_flyway_scripts_locations=getValue("DB_SCHEMA_FLYWAY_SCRIPTS_LOCATIONS","classpath:che-schema,classpath:codenvy-schema")
 
 ###############################
+# Rsync backup/restore configuration
+  $rsync_restore_bwlimit=getValue("RSYNC_RESTORE_BWLIMIT","7000")
```

Comment:
can you please add those three ENV vars
RSYNC_RESTORE_BWLIMIT
RSYNC_BACKUP_BWLIMIT
RSYNC_BACKUP_BWLIMIT

in codenvy.env with clear explanation of them.